client/src/store.ts

```diff
@@ -24,19 +24,20 @@ export type Create = (
   api: ApiHistory | ApiNoHistory,
   params?: { lastUpdatedParam: number; values: any[] },
   options?: any,
-) => Promise<number>
+) => Promise<number | number[]>
 export type Update = (
   api: ApiHistory | ApiNoHistory,
-  productId: number,
+  productId: number | number[],
```

Comment:
That change seems to lead to several incompatible API calls

![image](https://user-images.githubusercontent.com/50099643/208230450-d292a8d8-2064-434a-8028-d8aa24439e11.png)

Reply:
Oh right, forgot to call "yarn build" in buerli-examples. Will fix it.

Reply:
I've added a condition to check if product id is a single number. See [53f29de](https://github.com/awv-informatik/buerli-examples/pull/11/commits/53f29de58762e627aeac22d90d5f51fdb18b80e8)